tests/test_converter.py

```diff
@@ -1627,6 +1626,7 @@ def test_circuit_to_qiskit_register_size(self, operations, register_size):
         [([qml.PauliX(0), qml.PauliY(1)], "y"), ([[qml.CNOT([0, 1]), qml.Hadamard(1)], "h"])],
     )
     @pytest.mark.parametrize("measure", [True, False])
+    # pylint:disable=pointless-statement
```

Comment:
Why is this used? Maybe we could improve this test to skip this? 🤔

Reply:
Improved 😎